src/commands/dev-env-import-sql.ts

```diff
@@ -111,38 +120,44 @@ export class DevEnvImportSQLCommand {
 			fs.unlinkSync( resolvedPath );
 		}
 
-		const cacheArg = [ 'wp', 'cache', 'flush', '--skip-plugins', '--skip-themes' ].concat(
-			this.options.quiet ? '--quiet' : []
-		);
-		await exec( lando, this.slug, cacheArg );
+		await flushCache( lando, this.slug, this.options.quiet );
```

Comment:
Also more refactor by extraction here so that I could read this code more easily to understand what's going on.